packages/web3-errors/src/error_codes.ts

```diff
@@ -52,38 +52,39 @@ export const ERR_TX_OUT_OF_GAS = 406;
 export const ERR_RAW_TX_UNDEFINED = 407;
 
 export const ERR_TX_INVALID_SENDER = 408;
-export const ERR_TX_INVALID_CALL = 409;
-export const ERR_TX_MISSING_CUSTOM_CHAIN = 410;
-export const ERR_TX_MISSING_CUSTOM_CHAIN_ID = 411;
-export const ERR_TX_CHAIN_ID_MISMATCH = 412;
-export const ERR_TX_INVALID_CHAIN_INFO = 413;
-export const ERR_TX_MISSING_CHAIN_INFO = 414;
-export const ERR_TX_MISSING_GAS = 415;
-export const ERR_TX_INVALID_LEGACY_GAS = 416;
-export const ERR_TX_INVALID_FEE_MARKET_GAS = 417;
-export const ERR_TX_INVALID_FEE_MARKET_GAS_PRICE = 418;
-export const ERR_TX_INVALID_LEGACY_FEE_MARKET = 419;
-export const ERR_TX_INVALID_OBJECT = 420;
-export const ERR_TX_INVALID_NONCE_OR_CHAIN_ID = 421;
-export const ERR_TX_UNABLE_TO_POPULATE_NONCE = 422;
-export const ERR_TX_UNSUPPORTED_EIP_1559 = 423;
-export const ERR_TX_UNSUPPORTED_TYPE = 424;
-export const ERR_TX_DATA_AND_INPUT = 425;
-export const ERR_TX_POLLING_TIMEOUT = 426;
-export const ERR_TX_RECEIPT_MISSING_OR_BLOCKHASH_NULL = 427;
-export const ERR_TX_RECEIPT_MISSING_BLOCK_NUMBER = 428;
-
-export const ERR_TX_LOCAL_WALLET_NOT_AVAILABLE = 429;
-
-export const ERR_TX_NOT_FOUND = 430;
-export const ERR_TX_SEND_TIMEOUT = 431;
-export const ERR_TX_BLOCK_TIMEOUT = 432;
-
-export const ERR_TX_SIGNING = 433;
-export const ERR_TX_GAS_MISMATCH = 434;
-
-export const ERR_TX_CHAIN_MISMATCH = 435;
-export const ERR_TX_HARDFORK_MISMATCH = 436;
+export const ERR_TX_INVALID_RECEIVER = 409;
+export const ERR_TX_INVALID_CALL = 410;
+export const ERR_TX_MISSING_CUSTOM_CHAIN = 411;
+export const ERR_TX_MISSING_CUSTOM_CHAIN_ID = 412;
+export const ERR_TX_CHAIN_ID_MISMATCH = 413;
+export const ERR_TX_INVALID_CHAIN_INFO = 414;
+export const ERR_TX_MISSING_CHAIN_INFO = 415;
+export const ERR_TX_MISSING_GAS = 416;
+export const ERR_TX_INVALID_LEGACY_GAS = 417;
+export const ERR_TX_INVALID_FEE_MARKET_GAS = 418;
+export const ERR_TX_INVALID_FEE_MARKET_GAS_PRICE = 419;
+export const ERR_TX_INVALID_LEGACY_FEE_MARKET = 420;
+export const ERR_TX_INVALID_OBJECT = 421;
+export const ERR_TX_INVALID_NONCE_OR_CHAIN_ID = 422;
+export const ERR_TX_UNABLE_TO_POPULATE_NONCE = 423;
+export const ERR_TX_UNSUPPORTED_EIP_1559 = 424;
+export const ERR_TX_UNSUPPORTED_TYPE = 425;
+export const ERR_TX_DATA_AND_INPUT = 426;
+export const ERR_TX_POLLING_TIMEOUT = 427;
+export const ERR_TX_RECEIPT_MISSING_OR_BLOCKHASH_NULL = 428;
+export const ERR_TX_RECEIPT_MISSING_BLOCK_NUMBER = 429;
+
+export const ERR_TX_LOCAL_WALLET_NOT_AVAILABLE = 430;
+
+export const ERR_TX_NOT_FOUND = 431;
+export const ERR_TX_SEND_TIMEOUT = 432;
+export const ERR_TX_BLOCK_TIMEOUT = 433;
+
+export const ERR_TX_SIGNING = 434;
+export const ERR_TX_GAS_MISMATCH = 435;
+
+export const ERR_TX_CHAIN_MISMATCH = 436;
+export const ERR_TX_HARDFORK_MISMATCH = 437;
```

Comment:
This will be breaking change for many error codes, and we can avoid it by making` ERR_TX_INVALID_RECEIVER = 437` ?

Reply:
resolved. thanks